src/app/api/admin/[...slug]/route.js

```diff
@@ -11,9 +11,14 @@ const handler = withErrorHandler(
       throw new NotAuthorizedException();
     }
 
-    const json = await req.json();
-    const result = await defaultHandler(json, prisma);
-    return NextResponse.json(result);
+    try {
+      const json = await req.json();
+      const result = await defaultHandler(json, prisma);
+      return NextResponse.json(result);
+    } catch (err) {
+      const {message} = err;
+      return NextResponse.json({ error: { message } }, { status: 500 });
+    }
```

Comment:
🟡This looks like a redundant try catch statement, there is a withErrorHandler wrapper around this function that handles errors. Any particular reason to use it here?

Reply:
true, thank you